default/stringtables/en.txt

```diff
@@ -4705,6 +4705,29 @@ Important sections include [[encyclopedia CATEGORY_GAME_CONCEPTS]] and the recen
 
 '''
 
+SITREP_DISMISS_HIDE_TITLE
+Dismissing and Hiding SitReps
+
+SITREP_DISMISS_HIDE_SHORT_DESC
+Dismissing a specific SitRep and hiding SitRep types.
+
+SITREP_DISMISS_HIDE_TEXT
+'''There are two options for ignoring Situation Reports (SitReps): Dismissing and Hiding.
+
+Dismissing a SitRep will keep that specific SitRep for displaying.
+There are options to dismiss for a certain number of turns, or indefinitely.
+When dismissing a SitRep, it will only stay dismissed for the current game.
+If you restart the client or reload the current game, these SitReps will display again.
+Dismissing is always for a specific SitRep, such as a reminder to build a Gas Generator on planet Delphi II.
```

Comment:
Yes, meaning the exact same message.

In your scenario, if the sitrep was dismissed for X turns, it will still show on turn 10.
If it was Indefinitely Dismissed, it would hide that sitrep for all turns (changed to Completely Ignore, as indefinite is misleading).